lib/services/authentication_services.dart

```diff
@@ -0,0 +1,33 @@
+import '../model/user.dart';
+import '../services/login.dart';
+
+abstract class AuthenticationService {
+  Future<User> getCurrentUser();
+  Future<User> signInWithEmailAndPassword(String email, String password);
+  Future<void> signOut();
+}
+
+class FakeAuthenticationService extends AuthenticationService {
+  @override
+  Future<User> getCurrentUser() async {
+    return null; // return null for now
+  }
+
+  @override
+  Future<User> signInWithEmailAndPassword(String email, String password) async {
+    final Login api = Login();
+    await api.login(email, password).then((User user) {
+      print(user.token);
+    });
+
+    return User(email, password, user: 'test');
```

Comment:
this doesn't look like its done properly

Reply:
@manishiitg this's done as well